src/debugAdapter/goDebug.ts

```diff
@@ -268,9 +269,11 @@ class Delve {
 					connectClient(port, host);
 				}
 			});
+			let that = this;
```

Comment:
You can use a fat arrow function (`code => { ... }`) instead of `function`, then `this` will be correct